core/test/utils/urlUtils.js

```diff
@@ -0,0 +1,53 @@
+const _ = require('lodash');
+const sinon = require('sinon');
+const config = require('../../server/config');
+const UrlUtils = require('@tryghost/url-utils');
+const urlUtils = require('../../server/lib/url-utils');
+
+let sandbox;
+
+const getInstance = (options) => {
+    const opts = {
+        url: options.url,
+        adminUrl: options.adminUrl,
+        apiVersions: options.apiVersions,
+        slugs: options.slugs,
+        redirectCacheMaxAge: options.redirectCacheMaxAge,
+        baseApiPath: '/ghost/api'
+    };
+
+    return new UrlUtils(opts);
+};
+
+const stubUrlUtils = (options, sandbox) => {
+    const stubInstance = getInstance(options);
+
+    Object.keys(urlUtils).forEach((key) => {
+        sandbox.stub(urlUtils, key).callsFake(stubInstance[key]);
+    });
+};
+
+// Method for regressions tests must be used with restore method
+const stubUrlUtilsFromConfig = () => {
+    sandbox = sinon.createSandbox();
```

Comment:
Is there a reason we create a new sandbox every time this is called? I think you can create as `const sandbox` and then replace this line w/ `sandbox.restore()`.

Reply:
@allouis wasn't sure about behavior in this case, but looks like you are right, works perfectly with a single instance :+1: This simplified a code a bit, niceeee :smiley:

Reply:
Yay! :dancer: